README.md

```diff
@@ -1 +1,46 @@
-# bff-samples
\ No newline at end of file
+## Ballerina Persists module with databases
+
+Ballerina's persistence features offer a straightforward way to work with data sources, making it easier to build an application's data access layer. By simply defining the required Ballerina records, the persistence package auto-generates all data access functions. This eliminates repetitive code, providing a simplified interface for inserting, updating and querying data with data tables directly mapped to Ballerina records.
+
```

Comment:
Follow a consistent format for README.md file. For other two samples there was a subtitle named `Set up` , but here we don't have it.

Reply:
Added the `set up` section.